osclib/adi_command.py

```diff
@@ -58,6 +60,10 @@ def check_adi_projects(self):
             self.check_adi_project(p)
 
     def create_new_adi(self, wanted_requests, by_dp=False, split=False):
+        needed_expand_allpkgs = self.config.get('expand-allpkgs-from-same-srcprj')
```

Comment:
I am never a fan of mixing abbreviations, but perhaps we can at least drop "same" since it does not really apply. It's a whitelist of source projects from which to expand all packages.

Perhaps even, `expand-from-source-projects` or `source-projects-expand` or `...-to-expand`.

Similar rule could be applied to variable names as they are rather different.

Better still:

```python
source_projects_expand = self.config.get('source_projects_expand', '').split()
```

Then the rest of the code can just be dropped.